docs/_includes/generated/rocks_db_configurable_configuration.html

```diff
@@ -63,16 +63,16 @@
             <td>Tne maximum number of write buffers that are built up in memory. RocksDB has default configuration as '2'.</td>
         </tr>
         <tr>
-            <td><h5>state.backend.rocksdb.writebuffer.number-to-merge</h5></td>
+            <td><h5>state.backend.rocksdb.writebuffer.memorysize</h5></td>
```

Comment:
Seems that the order of config `state.backend.rocksdb.writebuffer.memorysize` and `state.backend.rocksdb.writebuffer.number-to-merge` have been changed, so there show the diff here for both config